### PR TITLE
Update Layout of Project Items to be Hyperlinked Images

### PR DESCRIPTION
**Pull request by amuamushu**
https://screenshot.googleplex.com/z9Mxq0AeYTt.png

Having each project item be an image would allow users to gauge the topic of the project before reading the entire project title. Hyperlinking the image allow users to easily see the projects that interest them.

### CODE REVIEW

**Comment by amuamushu** [comment]
top of page: https://screenshot.googleplex.com/iznN1hoKO3r.png
bottom of page: https://screenshot.googleplex.com/Y82woqJcNPk.png

Additional screenshot showing how it looks in a smaller window: https://screenshot.googleplex.com/yUtNaksGoRH.png

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h2>Background</h2>
       <br>
       <h2>Projects:</h2>
       <div id="projects">
-        <a href="https://github.com/amuamushu/cogs9_final_project" target="_blank">
+        <a href="https://drive.google.com/file/d/1Jb52Jk0HLoNbdVDg3z5VrQMrCsxFrkv-/view?usp=sharing" target="_blank">
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html:50]: Is this your personal drive? I am assuming yes.

Comment (amuamushu) [reply portfolio/src/main/webapp/index.html:50]: Yes, it is a link to a pdf in my personal drive. Should I move the file to my @google drive or remove it completely?

**Comment by amuamushu** [comment]
Fixes issue #4

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (billie-cat) [reply portfolio/src/main/webapp/index.html:50]: No, because once you leave you want this to still work :)

Comment (amuamushu) [reply portfolio/src/main/webapp/index.html:50]: Understood! 
Done

**Comment by billie-cat** [comment]
Please look at this doc on writing good CL descriptions: https://google.github.io/eng-practices/review/developer/cl-descriptions.html

I think you can add high level details in terms of what layout changes you made.

**Pull request merged**